src/compose.js

```diff
@@ -14,9 +14,9 @@ export default function compose(...funcs) {
     return arg => arg
   }
 
-  if (funcs.length === 1) {
-    return funcs[0]
+  if (funcs.length===1) {
+  return funcs[0]
```

Comment:
format: style mismatch:
```suggestion
    return funcs[0]
```

`2edfa193` Replace ⏎ with ⏎␣⁺␣⁺ in the beginning of line